docs/11-circleci/02-activation.mdx

```diff
@@ -7,6 +7,9 @@ Unity separates its licenses between `Personal` and `Professional`. The former a
 and the latter to the _Pro_ and _Plus_ versions. Based on your use case proceed to the
 [Personal](#personal-license) or [Professional](#professional-license) license section.
 
+> _**Note:** GameCI does not acquire nor store your Unity email, password or serial. They are
+> required for activating the license during the build and test steps._
+
```

Comment:
I'd argue that modern tools would never do this. I don't think this note is really needed.

Reply:
I don't mind removing it. It is a copy-paste from the builder: https://github.com/game-ci/documentation/blob/main/docs/03-github/04-builder.mdx?plain=1#L26.